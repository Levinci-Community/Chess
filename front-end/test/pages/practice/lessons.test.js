import React from "react";
import { render } from "@testing-library/react";
import LessonsPage from "../../../src/pages/practice/lessons";
import { BrowserRouter as Router } from "react-router-dom";

jest.mock("react-i18next", () => ({
    useTranslation: () => ({
      t: (key) => key,
    }),
  }));
  
  test("renders LessonsPage snapshot", () => {
    const { container } = render(
      <Router>
        <LessonsPage />
      </Router>
    );
    expect(container).toMatchSnapshot();
  });