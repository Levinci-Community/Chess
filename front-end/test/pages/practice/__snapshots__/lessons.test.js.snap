// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders LessonsPage snapshot 1`] = `
<div>
  <div
    class="chakra-container css-3laa41"
  >
    <h1
      class="chakra-heading css-1itv5e3"
    >
      Lessons
    </h1>
    <ul
      class="css-1fqirv5"
      role="list"
    >
      <p
        class="chakra-text css-0"
      >
        Loading...
      </p>
    </ul>
  </div>
</div>
`;
